bin/test-workflow/0_prep_env.sh

```diff
@@ -6,30 +6,37 @@ set -o pipefail
 export CONJUR_OSS_HELM_INSTALLED="${CONJUR_OSS_HELM_INSTALLED:-true}"
 export UNIQUE_TEST_ID="$(uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10)"
 
-# PLATFORM is used to differentiate between general Kubernetes platforms (K8s vs. oc), while
-# CONJUR_PLATFORM is used to differentiate between sub-platforms (for vanilla K8s, KinD vs. GKE) for the Conjur deployment
-# APP_PLATFORM serves the same purpose as CONJUR_PLATFORM, but for the test app deployment
+# PLATFORM is used to differentiate between general Kubernetes platforms (kubernetes, openshift), while
+# CONJUR_PLATFORM is used to differentiate between sub-platforms (kind, gke, jenkins, openshift) for the Conjur deployment
+# APP_PLATFORM serves the same purpose as CONJUR_PLATFORM, but for the test app deployment (kind, gke, openshift)
 if [[ "$CONJUR_OSS_HELM_INSTALLED" == "true" ]]; then
   CONJUR_PLATFORM="${CONJUR_PLATFORM:-kind}"
 else
   CONJUR_PLATFORM="${CONJUR_PLATFORM:-gke}"
 fi
 export CONJUR_PLATFORM
 
-if [[ "$CONJUR_PLATFORM" == "oc" ]]; then
-  PLATFORM="openshift"
+if [[ "$CONJUR_PLATFORM" == "openshift" ]]; then
+  PLATFORM="${PLATFORM:-openshift}"
 else
-  PLATFORM="kubernetes"
+  PLATFORM="${PLATFORM:-kubernetes}"
 fi
```

Comment:
In these cases, do we want to defer to the existing `PLATFORM` variable? Might allow for `CONJUR_PLATFORM` and `PLATFORM` to be out of phase, instead of aligning them.

Reply:
That's exactly why the change was made - for the flow being added `PLATFORM=openshift` and `CONJUR_PLATFORM=jenkins`